packages/client/hmi-client/src/page/Home.vue

```diff
@@ -163,14 +168,28 @@ import Dropdown from 'primevue/dropdown';
 import MultiSelect from 'primevue/multiselect';
 import SelectButton from 'primevue/selectbutton';
 import { useProjectMenu } from '@/composables/project-menu';
-import { Project } from '@/types/Types';
+import { Project, ClientEventType, ProgressState } from '@/types/Types';
 import { Vue3Lottie } from 'vue3-lottie';
 import EmptySeed from '@/assets/images/lottie-empty-seed.json';
 import TeraInputText from '@/components/widgets/tera-input-text.vue';
 import { FilterService } from 'primevue/api';
+import { useNotificationManager } from '@/composables/notificationManager';
 
 const { isProjectConfigDialogVisible, menuProject } = useProjectMenu();
 
+const { notificationItems } = useNotificationManager();
+
+const cloningProjects = computed(() => {
+	const items: any = [];
+	notificationItems.value.forEach((item) => {
+		if (item.type === ClientEventType.CloneProject && item.status === ProgressState.Running) {
+			const project = myFilteredSortedProjects.value.find((p) => p.id === item.assetId);
+			items.push(project);
+		}
+	});
+	return items;
+});
```

Comment:
How does `items.length` become 0 in order to trigger the refresh? Is there somewhere we would remove the "ClientEventType.CloneProject" item from notificationItems?

Reply:
Is there an edge case that I'm missing?

I have the computed triggering on the status change only because I'm using it to check if the process is still running inside of the computed.

Reply:
I am not sure, but it seems like you are depending on the notificationItems to change from zero => not_zero => zero. There doesn't seem to be anything to automatically "remove" stuff from notificationItems??

Reply:
I found what I am looking for - we update existing notifications as oppose to always appending new notifications.